plaso/cli/psteal_tool.py

```diff
@@ -186,9 +186,9 @@ def AnalyzeEvents(self):
     storage_reader = storage_factory.StorageFactory.CreateStorageReaderForFile(
         self._storage_file_path)
     if not storage_reader:
-      logger.error('Format of storage file: {0:s} not supported'.format(
-          self._storage_file_path))
-      return
+      raise errors.BadConfigOption(
```

Comment:
please update docstring as well

Reply:
This already has "BadConfigOption: when a configuration parameter fails validation." and the format of the file provided as an option is being checked - what do you want the in the docstring here?

Reply:
Maybe:
BadConfigOption: when a configuration parameter fails validation or the storage file cannot be opened with read access.

Reply:
Done